cadence/src/io.rs

```diff
@@ -77,6 +77,11 @@ where
         let required = buf.len() + self.line_ending.len();
 
         if required > self.capacity {
+            // if buffer non-empty,
```

Comment:
Can you put the comment on a single line with a period at the end.

Reply:
Fixed in https://github.com/56quarters/cadence/pull/218/commits/b83ab03ad32a142d2f0e5846bc9583ab3c75a399